src/repeating-form-accessor.ts

```diff
@@ -99,9 +99,12 @@ export class RepeatingFormAccessor<
 
   @computed
   get accessors(): RepeatingFormIndexedAccessor<D, G>[] {
+    // we get the entries in this map, in order of index
+    const length = Array.from(this.repeatingFormIndexedAccessors.values())
```

Comment:
neat change, it is indeed faster to get length first / in a separate variable, and use it later on